notebooks/ch11/gradient_descent_define.md

```diff
@@ -251,24 +251,24 @@ plot_one_gd_iter(pts, 14.49, mse, grad_mse)
 
 We now have the full algorithm for gradient descent:
 
-1. Choose a starting value of $ \hat{\theta_0} $ (0 is a common choice).
-2. Compute $ \hat{\theta_0} - \alpha \cdot \frac{\partial}{\partial \theta} L(\hat{\theta_0}, y) $ and store this as the new value of $ \theta $.
-3. Repeat until $ \hat{\theta_0} $ doesn't change between iterations.
+1. Choose a starting value of $ \theta $ (0 is a common choice).
+2. Compute $ \theta - \alpha \cdot \frac{\partial}{\partial \theta} L(\theta, y) $ and store this as the new value of $ \theta $.
+3. Repeat until $ \theta $ doesn't change between iterations.
 
 You will more commonly see the gradient $ \nabla_\theta $ in place of the partial derivative $ \frac{\partial}{\partial \theta} $. The two notations are essentially equivalent, but since the gradient notation is more common we will use it in the gradient update formula from now on:
 
 $$
-\hat{\theta_{t+1}} = \hat{\theta_t} - \alpha \cdot \nabla_\theta L(\hat{\theta_t}, y)
+\theta^{t+1} = \theta^t - \alpha \cdot \nabla_\theta L(\theta^t, y)
 $$
 
 To review notation:
 
-- $ \hat{\theta_t} $ is the current choice of $ \hat{\theta_0} $.
-- $ \hat{\theta_{t+1}} $ is the next choice of $ \hat{\theta_0} $.
-- $ \alpha $ is called the learning rate, usually set to a small constant. Sometimes it is useful to start with a larger $ \alpha $ and decrease it over time. If $ \alpha $ changes between iterations, we use the variable $ \alpha_t $ to mark that $ \alpha $ varies over time $ t $.
-- $ \nabla_\theta L(\hat{\theta_0}, y) $ is the partial derivative / gradient of the loss function at $ \hat{\theta_0} $.
+- $ \theta^t $ is the current choice of $ \theta $.
+- $ \theta_{t+1} $ is the next choice of $ \theta $.
+- $ \alpha $ is called the learning rate, usually set to a small constant. Sometimes it is useful to start with a larger $ \alpha $ and decrease it over time. If $ \alpha $ changes between iterations, we use the variable $ \alpha^t $ to mark that $ \alpha $ varies over time $ t $.
+- $ \nabla_\theta L(\theta, y) $ is the partial derivative / gradient of the loss function at $ \theta $.
```

Comment:
I think we should sync this one up with the review in gradient_stochastic. I like how the one in gradient_stochastic mentions "current estimate of θ∗ at the tth iteration"

I think we can actually have the gradient bullet point match what is displayed above, and say it is the partial derivative / gradient of the loss function with respect to theta at theta^t